manpages/mkfs.fat.8.in

```diff
@@ -20,176 +20,174 @@
 1.2M, 1.44M, 2.88M), a 16 bit FAT otherwise.
-This can be overridden with the \fB\-F\fR option.
+This can be overridden with the \fB\-F\fP option.
 Some PC-specific boot sector fields aren't written, and a boot message (option
-\fB\-m\fR) is ignored.
+\fB\-m\fP) is ignored.
 .\" ----------------------------------------------------------------------------
 .SH OPTIONS
-.IP "\fB\-a\fR" 4
-Normally, for any filesystem except very small ones, \fBmkfs.fat\fR will align
+.IP "\fB\-a\fP" 4
+Normally, for any filesystem except very small ones, \fBmkfs.fat\fP will align
 all the data structures to cluster size, to make sure that as long as the
 partition is properly aligned, so will all the data structures in the
 filesystem.
 This option disables alignment; this may provide a handful of additional
 clusters of storage at the expense of a significant performance degradation on
 RAIDs, flash media or large-sector hard disks.
-.IP "\fB\-A\fR" 4
+.IP "\fB\-A\fP" 4
 Select using the Atari variation of the FAT filesystem if that isn't active
 already, otherwise select standard FAT filesystem.
-This is selected by default if \fBmkfs.fat\fR is run on 68k Atari Linux.
-.IP "\fB\-b\fR \fISECTOR-OF-BACKUP\fR" 4
+This is selected by default if \fBmkfs.fat\fP is run on 68k Atari Linux.
+.IP "\fB\-b\fP \fISECTOR-OF-BACKUP\fP" 4
 Selects the location of the backup boot sector for FAT32.
 Default depends on number of reserved sectors, but usually is sector 6.
 The backup must be within the range of reserved sectors.
 .IP "\fB\-c" 4
 Check the device for bad blocks before creating the filesystem.
-.IP "\fB\-C\fR" 4
-Create the file given as \fIDEVICE\fR on the command line, and write the
+.IP "\fB\-C\fP" 4
+Create the file given as \fIDEVICE\fP on the command line, and write the
 to-be-created filesystem to it.
 This can be used to create the new filesystem in a file instead of on a real
-device, and to avoid using \fBdd\fR in advance to create a file of appropriate
+device, and to avoid using \fBdd\fP in advance to create a file of appropriate
 size.
-With this option, the \fIBLOCK-COUNT\fR must be given, because otherwise the
+With this option, the \fIBLOCK-COUNT\fP must be given, because otherwise the
 intended size of the filesystem wouldn't be known.
 The file created is a sparse file, which actually only contains the meta-data
 areas (boot sector, FATs, and root directory).
 The data portions won't be stored on the disk, but the file nevertheless will
 have the correct size.
 The resulting file can be copied later to a floppy disk or other device, or
 mounted through a loop device.
-.IP "\fB\-D\fR \fIDRIVE-NUMBER\fR" 4
+.IP "\fB\-D\fP \fIDRIVE-NUMBER\fP" 4
 Specify the BIOS drive number to be stored in the FAT boot sector.
-This value is usually 0x80 for hard disks and 0x00 for floppy devices or
-partitions to be used for floppy emulation.
-.IP "\fB\-f\fR \fINUMBER-OF-FATS\fR" 4
+.IP "\fB\-f\fP \fINUMBER-OF-FATS\fP" 4
```

Comment:
You deleted some text here.

Reply:
Restored.

Reply:
OUCH, it looks like you force-pushed at least one commit, which means it is no longer possible to look at just the most recent changes, without reviewing *EVERYTHING* all over again.  Consequently, i'm giving up on this review.  Besides, code review on github is extremely painful in the first place, i absolutely hate the interface.  It is so unusable!

Reply:
I have just restored this missing paragraph and fixed name for WP ROOT. SF file. Nothing more. All other review comments are not processed yet, there are lot of stuff which needs to be changed. Diff what was changed is here: https://github.com/dosfstools/dosfstools/compare/f1badbcffb8d3834aaf02cdd1e4bfb2667b428ea..b4e6f9b031007d2d32b300df758c7b27d9e6fe3b